data/korath/korath missions.txt

```diff
@@ -257,3 +257,571 @@ mission "Small Scale Delivery"
+				launch
+
+			label "third attempt"
+			scene scene/blue-whale
+			action
+				set "landed on Rabasot Pa'ah"
+				log "Discovered a creature in a bay on the ocean world of Rabasot Pa'ah with the front half of a whale and the back half of a giant squid. They seem out-of-place, since the planet is almost devoid of macroscopic lifeforms. Perhaps someone is shipping them to the planet?"
+			`On your third attempt, you dodge most of the storms by flying into the atmosphere at an odd angle that keeps you in stable air nearly the whole way. After a long flight through safe atmosphere, you finally reach the bay and land on a durable mesa overlooking it. Extending your ship's artificial gravity field to the surface, you don your suit and walk out. It quickly becomes clear the mesa is artificial: it's a constructed landing pad for a ship of vast scale.`
+			`	This is the perfect location for life, so it's no surprise there are small plants and a few fish in the water. The seas are calmer here than the rest of the planet and an unusual formation in the ionosphere protects this spot from the worst rays of the nova. The dominant land-based lifeform seems to be toothed worms about the size of your arm.`
+			`	The strangest feature is what appears to be slow-moving islands. As one comes closer, its form is more clear: an unholy marriage between a blue whale and a giant squid. On the far end of the mesa, you spy a gravity lift system that looks suitable for lifting such a beast out of the water.`
+				decline
+
+
+
+mission "Korath [F04]: Fishy Reasoning"
+	landing
+	name "Seek answers from Iris"
+	description "Visit Iris at <planet> and ask her about Korath, fish transfers, an ocean planet, and her rainbow feather."
+	clearance "You explain the purpose of your visit, and the authorities inform you that Iris will expect you."
+	source
+		not planet "La'ab Bamen"
+	destination "La'ab Bamen"
+	to offer
+		has "First Contact: Korath Exiles: offered"
+
+		# Require a successful landing on Rabasot Pa'ah, since that gives you the whale picture.
+		has "landed on Rabasot Pa'ah"
+
+		# You have to meet Iris and know she has knowledge of Korath:
+		has "Korath [F02]: A Fishy Proposition: done"
+	on offer
+		conversation
+			`Suddenly, you have a feeling like a thousand piece jigsaw puzzle just came together before your eyes. When the Korath were drawing systems, three seemed to jump out at you. Misam, home of New Tortuga; Atik, where the La'ab Bamen black market resides; and Kor Ak'Mari, the ocean planet's system. The first fish you transferred from New Tortuga to La'ab Bamen looked like a tiny relative of the island-sized fish on the ocean planet. She had quite a lot of knowledge of the Korath, and spoke of them with admiration.`
```

Comment:
I rewrote this so that a few logs catch Captain Last's eye while reviewing damage and transit logs. The logs are the same ones as before, and the mission has the same result. They're less pushy about going to the destination.

How does it look?

Reply:
You may need to reload the page. I just reloaded and the changes are definitely there.

> `While skimming damage and transit logs from your recent trips, a few old log entries catch your eye.`
> `    When the Korath were drawing systems, three seemed to jump out at you. You now know they're Misam, home of New Tortuga; Atik, where the La'ab Bamen black market resides; and Kor Ak'Mari, the ocean planet's system. The first fish you transferred from New Tortuga to La'ab Bamen looked like a tiny relative of the island-sized fish on the ocean planet.`
> `    Iris had quite a lot of knowledge of the Korath, and spoke of them with admiration. Most curious of all is the rainbow feather on her necklace. It was the exact shape of feathers on the Korath of Far'en Lei, but far more colorful.`
> `    You last saw Iris on <planet>. Perhaps it is time to seek answers.`

Reply:
I think there should be a timed event with a random period. It'll have the effect you describe, but it'll also ensure a reasonable delay.

Reply:
The deed is done. You have to land anywhere and then there is a delay of five to fourteen days. After that, you get the message on landing. That ensures you won't get the message immediately if you take two weeks to land.